docs/source/cpp/compute.rst

```diff
@@ -343,7 +343,7 @@ equivalents above and reflects how they are implemented internally.
 +-------------------------+---------+------------------------------------+------------------------+----------------------------------+-----------+
 | hash_count              | Unary   | Any                                | Int64                  | :struct:`CountOptions`           | \(2)      |
 +-------------------------+---------+------------------------------------+------------------------+----------------------------------+-----------+
-| hash_count_all          | Unary   |                                    | Int64                  |                                  |           |
+| hash_count_all          | Nullary |                                    | Int64                  |                                  |           |
```

Comment:
You're right. What i saw is that hash_count_all's constructor use unary, but it's unnecessary.